src/Actions/Order/CreateOrder.php

```diff
@@ -120,14 +120,15 @@ protected function prepareForValidation(): void
         $this->data['agent_id'] = $this->data['agent_id'] ?? $contact->agent_id;
         $this->data['approval_user_id'] ??= $contact->approval_user_id;
         $this->data['contact_bank_connection_id'] ??= $contact->contactBankConnections()->first()?->id;
-        $this->data['payment_target'] ??= $contact->payment_target_days
-            ?? $paymentType->payment_target
-            ?? 0;
         $this->data['payment_discount_target'] ??= $contact->discount_days
             ?? $paymentType->payment_discount_target
             ?? 0;
         $this->data['payment_discount_percent'] ??= $contact->discount_percent
-            ?? $paymentType->payment_discount_percent
+            ?? $paymentType->payment_discount_percentage
+            ?? 0;
+        $this->data['payment_target'] ??= $contact->payment_target_days
+            ?? $paymentType->payment_target
+            ?? ($this->data['payment_discount_target'] ? $this->data['payment_discount_target'] + 1 : null)
```

Comment:
why not 0 instead of null? Then you can remove the last ?? 0. Or do you expect that other conditions may be following in the future?